docs/docker.md

```diff
@@ -5,9 +5,41 @@
 A docker image for the Promscale Connector is available
 on Docker Hub at [timescale/promscale](https://hub.docker.com/r/timescale/promscale/).
 
+To fetch the latest Promscale image, please refer to the [releases](https://github.com/timescale/promscale/releases)
+section of Promscale github repository.
+
 A docker image of TimescaleDB with the `promscale`
 extension is available at on Docker Hub at
-[`timescaledev/promscale-extension:latest-pg12`](https://hub.docker.com/r/timescaledev/promscale-extension).
+[`timescaledev/promscale-extension:latest-pg12`](https://hub.docker.com/r/timescaledev/promscale-extension). You can
+install this via `docker pull timescaledev/promscale-extension:latest-pg12`.
+
+**Example using plain docker:**
+
+Install and run TimescaleDB with Promscale extension:
+```dockerfile
+docker run --name timescaledb -e POSTGRES_PASSWORD=<password> -d -p 5432:5432 timescaledev/promscale-extension:latest-pg12 postgres -csynchronous_commit=off
+```
+
+Get IP address of the above container which would be supplied as `db-host` flag to Promscale:
```

Comment:
I don't think this is the right way to connect two containers in docker. I think using a bridge network that supports name-based resolution is the way to go https://docs.docker.com/network/network-tutorial-standalone/#use-user-defined-bridge-networks (look for "automatic service discovery")